sdk/storage/azure_storage_blob/CHANGELOG.md

```diff
@@ -4,12 +4,28 @@
 
 ### Features Added
 
+- Added support for client construction directly from URLs:
+  - `AppendBlobClient::from_url()`
+  - `BlobClient::from_url()`
+  - `BlobContainerClient::from_url()`
+  - `BlobServiceClient::new()` (unchanged)
+  - `BlockBlobClient::from_url()`
+  - `PageBlobClient::from_url()`
+- Consequently, this also includes support for client construction directly from URLs containing SAS (shared access signature) query parameters.
```

Comment:
Sorry I missed this before. Let's make this even more clear that we added SAS support by simply saying. "Added support for SAS (shared access signature) URLs via the new `from_url` methods." As it is worded, it is kind of easy to miss.